app/src/pages/inside/filtersPage/filterTable/filterTableItem/filterTableItem.scss

```diff
@@ -200,6 +200,11 @@
     padding-right: 0;
   }
 }
+.delete-button {
+  background: none;
+  outline: none;
+  border: none;
```

Comment:
Use this  https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/ order for css rules

Reply:
Already not actual.